Mobile Buy SDK/Mobile Buy SDK/Models/Persistent/BUYProduct.m

```diff
@@ -118,12 +118,15 @@ - (BOOL)isDefaultVariant
 {
 	if ([self.variants count] == 1) {
 		BUYProductVariant *productVariant = [self.variants firstObject];
-		BUYOptionValue *optionValue = [productVariant.options anyObject];
-		NSString *defaultTitleString = @"Default Title";
-		NSString *defaultString = @"Default";
-		if ([productVariant.title isEqualToString:defaultTitleString] &&
-			([optionValue.value isEqualToString:defaultTitleString] || [optionValue.value isEqualToString:defaultString])) {
-			return YES;
+		if ([productVariant.options count] == 1) {
+			BUYOptionValue *optionValue = [productVariant.options anyObject];
+			NSString *defaultTitleString = @"Default Title";
+			NSString *defaultString = @"Default";
+			NSString *titleString = @"Title";
+			if ([optionValue.name isEqualToString:titleString] &&
```

Comment:
There's a flaw in this logic. The `optionValue.name` cannot be simultaneously equal to two different strings.